src/main/java/com/rultor/agents/req/Decrypt.java

```diff
@@ -115,4 +134,18 @@
         return commands;
     }
 
+    /**
+     * Proxy usage command creation.
+     * @param host Host.
+     * @param port Port.
+     * @return Command string.
+     */
+    private String createProxyString(final String host, final int port) {
```

Comment:
see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#method-names